RELEASE.md

```diff
@@ -61,6 +61,10 @@ Wait for the evergreen version to finish, and ensure that the release task compl
 Check that the version just released is available in the [Sonatype Nexus Repo Manager](https://oss.sonatype.org/#nexus-search;quick~mongodb-jdbc).  
 The release artifacts should appear on [Maven Central](https://search.maven.org/search?q=g:org.mongodb%20AND%20a:mongodb-jdbc) after a while.
 
+#### Notify the Web team about the new release
+Create a ticket through the [service desk](https://jira.mongodb.org/plugins/servlet/desk/portal/61) and request the link to the Maven download on the Download center page to be updated.  
```

Comment:
```suggestion
Create a ticket through the [service desk](https://jira.mongodb.org/plugins/servlet/desk/portal/61) and request the link to the Maven download on the Download Center Page to be updated.  
```
Should this be capitalized? It could make it a little more clear.

Also, is there a specific request type we want them to use because we could link them straight to that instead of just the general request portal (i.e., if we want them to always select _Web Production Request - New Web Page/ Update Existing Web Page_, we could make this the link: https://jira.mongodb.org/plugins/servlet/desk/portal/61/create/926).

Reply:
Yes, that totally makes sense! I'll update this. The link in the doc was the one they gave me, but it is better to remove any doubt on which section to use. 👍